CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ It is based on Tidepool Blip 1.27.
 ## 3.3.0-beta - 2024-02-14
 ### Changed
 - YLP-2851 Remove pump cartridge expiration date from Device view and PDF report
+- YLP-2850 Add product name (model) in pump information card
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏We usually order the tickets by number